engine/src/mac-core.mm

```diff
@@ -27,6 +27,8 @@
 
 #include "libscript/script.h"
 
+#include "uidc.h"
```

Comment:
This shouldn't include uidc. That's a level above MCPlatform - you need to define a separate PlatformSystemAppearance enum and map.